modules/representation_management/app/controllers/representation_management/v0/pdf_generator_2122a_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    class PdfGenerator2122aController < RepresentationManagement::V0::PdfGeneratorBaseController
+      # service_tag 'lighthouse-veteran' # Is this the correct service tag?
+
+      def create
+        # We'll need a process here to check the params to make sure all the
+        # required fields are present. If not, we'll need to return an error
+        # with status: :unprocessable_entity.  If all fields are accounted for
+        # we need to fill out the 2122 PDF with the data and return the file
+        # to the front end.
+
+        # This work probably belongs in the PDF Generation ticket.
+        render json: {}, status: :unprocessable_entity
+      end
+
+      private
+
+      def form_params
+        params.permit(all_params)
```

Comment:
Please see my comment about the shape of the params hash in the `modules/representation_management/app/controllers/representation_management/v0/pdf_generator_base_controller.rb` file below.